lib/developer_portal/config/initializers/liquid.rb

```diff
@@ -1,9 +1,6 @@
 # frozen_string_literal: true
 require 'liquid'
 
-# allow calling present? in {% if %}
```

Comment:
Now this is handled with `liquid_present_literal.rb` patch. In the new version `Liquid::Expression::LITERALS` is [frozen](https://github.com/Shopify/liquid/blob/v5.4.0/lib/liquid/expression.rb#L11).